runtime/ddr/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ target_include_directories(j9ddr_misc
 	${omr_SOURCE_DIR}/compiler
 )
 
-if(OMR_TOOLCONFIG STREQUAL "gnu")
+if((OMR_TOOLCONFIG STREQUAL "gnu") AND (NOT CMAKE_CXX_COMPILER_ID MATCHES Clang) )
```

Comment:
https://github.com/eclipse/openj9/blob/4baf3d7e44ab1b7d580d0097651e1e4908145c53/runtime/ddr/module.xml#L405-L414

I'm having some trouble determining if these two things are equivalent or if something is being left out.  The case I'm most uncertain of is newer xlc compilers that are clang based.  They tend to report themselves as gnu, clang, and xlc.

Reply:
From the omr side we set OMR_TOOLCONFIG=gnu for both gcc and clang (since most of the time its treated the same as gcc). CMake should not identify xlc as clang, however that was a bug until 3.11 (https://gitlab.kitware.com/cmake/cmake/issues/17784)

Reply:
Which version of cmake do we have deployed to our machines?

Reply:
The aix machines appear to have 3.14